BottomsUp/app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDataIntegrationTest.kt

```diff
@@ -17,19 +22,26 @@ class DrinkDataIntegrationTest {
 
     var drinkService = mockk<DrinkService>()
 
+    @Before
+    fun setUp()
+    {
+        mvm = MainViewModel()
+    }
+
```

Comment:
Used the @Before to create a new MainViewModel before every test. Before, creating a mvm was being called as a given for every test. Removes a bit of redundancy.

Reply:
Hmmm, good idea.